ioctl/cmd/account/accountlist.go

```diff
@@ -44,7 +45,8 @@ var accountListCmd = &cobra.Command{
 }
 
 type listMessage struct {
-	Accounts []account `json:"accounts"`
+	Accounts    []account `json:"accounts"`
+	Sm2Accounts []account `json:"sm2Accounts"`
```

Comment:
why we need to distinguish?

Reply:
> why we need to distinguish?

ioctl can handle different keys with different cryptography algorithms, if user stores those two kinds of key in same place (using same ioctl), ioctl need distinguish them when listing them for user.